README.md

```diff
@@ -21,27 +21,63 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
         with:
+          # The token of the user that should perform the merges. 
+          # This must be a personal access token with the necessary permissions
+          token: ${{ secrets.DEV_PUSH_TOKEN }}
           fetch-depth: 0
 
-      - uses: staffbase/autodev-action@latest
+      - name: Autodev
+        uses: staffbase/autodev-action@v1.0.0
         with:
+          # Base branch. This branch is used as base, in which other branches will be merged into.
+          # Default: master
+          base: main
+          # Weather the action should perform an "optimistic" merge of the given Pull requests.
+          # If this is set to false, the dev branch will only be build if all Pull requests can be merged.
+          # Default: false
           optimistic: true
+          # The github action creates a new comment inside every pull request.
+          # If you don't want any comments, you can disable the comments by setting this to true
+          # Default: false
           disableComments: false
+          # The token used to fetch the pull requests from the API
           token: ${{ secrets.DEV_PUSH_TOKEN }}
+          # E-mail address of the user that should merge the PRs
           user: ${{ secrets.DEV_PUSH_USER }}
 ```
 
-## Development
+# Release a new version of this action
```

Comment:
I would skip this section because we only need to add a new tag and no contributer will do this by his/her own.